pkgs/applications/gis/openorienteering-mapper/default.nix

```diff
@@ -0,0 +1,66 @@
+{ stdenv, fetchFromGitHub, gdal, cmake, ninja, proj, clipper, zlib, qtbase, qttools
+  , qtlocation, qtsensors, doxygen, cups, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  name = "OpenOrienteering-Mapper-${version}";
```

Comment:
@mpickering

[13.2. Package naming](https://nixos.org/nixpkgs/manual/#sec-package-naming):

> The name attribute must not contain uppercase letters — e.g., "mplayer-1.0rc2" instead of "MPlayer-1.0rc2".

Reply:
The rules are still evolving and I feel like this one is being phased out these days.